reports.html

```diff
@@ -1,25 +0,0 @@
----
```

Comment:
this is still needed.  It has incoming links to it from other sites.

It could be a redirect to the students tabs.

Reply:
Yeah sure! We can check for year 2019 too. And, will try to add a redirect on it.